packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,84 @@
+import Tippy from "@tippyjs/react";
+import clsx from "clsx";
+import * as React from "react";
+import styles from "./Tooltip.module.css";
+
+// Full list of Tippy props: https://atomiks.github.io/tippyjs/v6/all-props/
+type TooltipProps = {
+  /**
+   * The trigger element the tooltip appears next to.
+   */
+  children?: React.ReactElement;
+  /**
+   * The trigger element the tooltip appears next to.
+   *
+   * Use this instead of `children` if the trigger element is being
+   * stored in state or a ref. Most cases will use `children` and not
+   * `reference`.
+   */
+  reference?: React.RefObject<Element> | Element;
```

Comment:
This was `React.Node` in traject, but `React.ReactNode` not compatible with `TippyProp`'s reference: `React.RefObject<Element> | Element | null`.